ui/js/actions/app.js

```diff
@@ -202,6 +201,27 @@ export function doCheckUpgradeAvailable() {
   };
 }
 
+export function doCheckDaemonVersion() {
+  return function(dispatch, getState) {
+    lbry.version().then(({ lbrynet_version }) => {
+      if (config.lbrynetDaemonVersion != lbrynet_version) {
+        dispatch({
+          type: types.OPEN_MODAL,
+          data: {
+            modal: "incompatibleDaemon",
+          },
+        });
+      }
+    });
+  };
+}
+
+export function doSkipWrongDaemonNotice() {
```

Comment:
I'm inclined to just not allow this to be skipped. If the wrong daemon version is running, something went wrong with your install.